venafi/provider.go

```diff
@@ -45,12 +45,20 @@ Example for Venafi Cloud: Default`,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("VENAFI_USER", nil),
 				Description: `WebSDK user for Venafi Platform. Example: admin`,
+				Deprecated:  ", please use access_token instead",
 			},
 			"tpp_password": &schema.Schema{
 				Type:        schema.TypeString,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("VENAFI_PASS", nil),
 				Description: `Password for WebSDK user. Example: password`,
+				Deprecated:  ", please use access_token instead",
+			},
+			"access_token": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("VENAFI_ACCESS", nil),
```

Comment:
Environment variable should be called "VENAFI_TOKEN"

Reply:
changed